lambdas/drivingpermitcheck/src/main/java/uk/gov/di/ipv/cri/drivingpermit/api/gateway/HttpRetryer.java

```diff
@@ -52,6 +66,7 @@ CloseableHttpResponse sendHTTPRequestRetryIfAllowed(HttpPost request)
 
             } catch (IOException e) {
                 if (!(e instanceof HttpConnectTimeoutException)) {
+                    eventProbe.log(Level.ERROR, e).counterMetric(THIRD_PARTY_REQUEST_SEND_FAIL);
                     throw e;
```

Comment:
Where does this throw to are we sure were not double logging

Reply:
I have removed the logging, it would have been double logged in verifyIdentity